### PR TITLE
chore(eslint): Store custom eslint-plugin locally

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: tools/eslint/package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@renovatebot/eslint-plugin",
+  "version": "0.0.5",
```
Comment (rarkins) [inline tools/eslint/package.json:3]: Should we just reset it to 0.0.0 or 1.0.0 or something "placeholder" like that?

Comment (zharinov) [reply tools/eslint/package.json:3]: It doesn't play any role to me 🤷

Comment (zharinov) [inline tools/eslint/package.json:3]:
```suggestion
  "version": "0.0.0",
```

**Review by viceice — DISMISSED** [review]

File: tools/eslint/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (zharinov) [reply tools/eslint/index.js:1]: Turned out, it doesn't :(

**Review by viceice — APPROVED** [review]

**Review by rarkins — APPROVED** [review]

**Pull request merged**